hibernate5/src/main/java/com/baeldung/hibernate/MultiTenantHibernateTest.java

```diff
@@ -0,0 +1,74 @@
+package com.baeldung.hibernate;
```

Comment:
We had removed this file in an earlier commit. It must have slipped back in when you switched machines.